src/main/java/com/rybka/model/CurrencyResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.rybka.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class CurrencyResponse {
```

Comment:
I think CurrencyData would be better here. 
*Response looks like some response from the API or another server from the first glance. 

In case, CurrencyData I understand that it's model with data returner from the internal methods.

Reply:
fixed